shared/components/initialScene.js

```diff
@@ -0,0 +1,184 @@
+import React, {Component, PropTypes} from 'react';
+import {Image, Platform, View, Text, Dimensions, TouchableOpacity} from 'react-native';
+import Swiper from 'react-native-swiper';
+import { Actions } from 'react-native-router-flux';
+
```

Comment:
this does not trigger eslint warning.
also, this is intended to separate importing library or importing images and others